src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -639,7 +640,15 @@ event::ptr primitive_inst::realloc_if_needed() {
                                         << fc_input_layout.to_short_string() << " to meet the input buffer alignment requirements for FC\n";
                     updated_layouts[dep_idx] = fc_input_layout;
                 }
+
+                // dynamic quantization is only applied to activation of FC
+                if (get_node().is_type<dynamic_quantize>()) {
+                    auto dyn_quan_scale_layout = dynamic_quantize_inst::__calc_output_layouts<ov::PartialShape>(updated_layouts[dep_idx], 0);
```

Comment:
I'm confused by this code. Can it be replaced with the generic check that node goes to 0 in port of FC instead? Or it's doing something different?

Reply:
Not sure what you mean by this comment. This code is to apply fake quantization into scale port too.

Reply:
Why do we need it?

Reply:
oops, not fake quantization, but fake alignment.

In a subgraph like `dynamic_quantize[out0, out1] --> fully_connected`, out0 will be fake-aligned. That means, fully_connected actually runs with fake-aligned size. We need to have aligned size for out1, too.